crates/ruff_linter/src/rules/flake8_bugbear/rules/cached_instance_method.rs

```diff
@@ -58,6 +58,13 @@ use crate::checkers::ast::Checker;
 ///         return square(self.value)
 /// ```
 ///
+/// ## Options
+///
+/// This rule only applies to regular methods, not static or class methods.
```

Comment:
It wasn't immediately clear to me why this is relevant for the listed options. Maybe say something that: You can use the following options to customize how Ruff classifies methods